.github/workflows/publish-latest.yml

```
@@ -21,9 +24,8 @@ jobs:
       - name: Create Release Pull Request or Publish to npm
         id: changesets
         uses: changesets/action@b98cec97583b917ff1dc6179dd4d230d3e439894
-        # with:
-        # version: yarn version:latest # commented out until we're ready for GA release
-        # publish: yarn release:latest # commented out until we're ready for GA release
+        with:
+          publish: yarn release:latest
```

Sorry, I don't see a `release:latest` command?

Only `version:latest` and `publish:latest`

---

Oh god, you're right. I need to take a breath 😄 
```suggestion
          publish: yarn publish:latest
```